documentation/aircraft-commands.md

```diff
@@ -0,0 +1,379 @@
+# Aircraft Command Reference
+
+## Table of Contents
+
+[Departure Commands](#departure-commands)
+- [Cleared as Filed](#cleared-as-filed)
+- [Climb via SID](#climb-via-sid)
+- [Takeoff](#takeoff)
+- [Taxi](#taxi)
+
+[Arrival Commands](#arrival-commands)
+- [Descend via STAR](#descend-via-star)
+- [Land](#land)
+
+[Routing Commands](#routing-commands)
+- [~~Fix~~](#fix)
+- [Hold](#hold)
+- [Proceed Direct](#proceed-direct)
+- [Route](#route)
+- [Reroute](#reroute)
+- [SID](#sid)
+- [STAR](#star)
+
+['Basic Control Instruction' Commands](#'basic-control-instruction'-commands)
+- [Altitude](#altitude)
+- [Heading](#heading)
+- [Speed](#speed)
+
+[Aircraft Query Commands](#aircraft-query-commands)
+- [Say Altitude](#say-altitude)
+- [Say Assigned Altitude](#say-assigned-altitude)
+- [Say Heading](#say-heading)
+- [Say Assigned Heading](#say-assigned-heading)
+- [Say Speed](#say-indicated-airspeed)
+- [Say Assigned Speed](#say-assigned-speed)
+
+[Miscellaneous Commands](#miscellaneous-commands)
+- [~~Abort~~](#abort)
+- [Squawk](#squawk)
+
+[System Commands](#system-commands)
+- [Airport](#airport)
+- [Move Data Block](#move-data-block)
+- [Pause](#pause)
+- [Timewarp](#timewarp)
+- [Tutorial](#tutorial)
```

Comment:
There's a tutorial command?  Do we really need that?

Reply:
lol NOPE. In my opinion, none of the system commands are remotely important except I guess timewarp, which we could later do a different way anyway.